onnx_caffe2/__init__.py

```diff
@@ -2,5 +2,3 @@
 from __future__ import division
 from __future__ import print_function
 from __future__ import unicode_literals
-
-from .version import version as __version__
```

Comment:
Based on the discussion in onnx, you probably don't want to do this.

Reply:
Yeah, will update the PR soon. Thanks. :-)